.github/workflows/snapshot-tests.yml

```diff
@@ -30,18 +30,22 @@ jobs:
         with:
           python-version: 3.12
 
+      - uses: pnpm/action-setup@v4
+        with:
+          version: 9.2.0
+
       -  name: Atom, cdxgen, sdkman, custom-json-diff installs
+         shell: bash
+         env:
+           SHELL: bash
          run: |
-           corepack enable
-           cdxgen_version=$(corepack pnpm pack | tail -1)
-           corepack pnpm install -g "$cdxgen_version"
+           cdxgen_version=$(pnpm pack | tail -1)
```

Comment:
corepack pnpm pack

Reply:
@prabhu `corepack pnpm pack` did not work as it tried to install from the npm registry rather than the packed file for some reason. This did work.

![image](https://github.com/CycloneDX/cdxgen/assets/80227828/7a6dc65c-e2db-4717-9db5-62ae9b1e7220)